models/repo/repo.go

```diff
@@ -686,11 +688,17 @@ func ComposeSSHCloneURL(doer *user_model.User, ownerName, repoName string) strin
 	return fmt.Sprintf("%s@%s:%s/%s.git", sshUser, sshHost, url.PathEscape(ownerName), url.PathEscape(repoName))
 }
 
+// ComposeTeaCloneLink returns Tea CLI clone command based on the given owner and repository name.
+func ComposeTeaCloneLink(ctx context.Context, owner, repo string) string {
```

Comment:
It is not a link, but a "command"

Reply:
I chose this name because everything is then stored in a `CloneLink{}`, to keep names consistent. But I can rename it to `ComposeTeaCloneCommand` if you prefer (but then it will still be stored in CloneLink)

Reply:
`ComposeTeaCloneCommand` seems better.

It's fine to store it in `CloneLink` at the moment.